pkg/condition/conditions.go

```diff
@@ -51,10 +82,22 @@ func (c *condition) WaitTillPodDataStoreSynced() {
 	}
 }
 
-// TODO: Add implementation later
 func (c *condition) IsWindowsIPAMEnabled() bool {
-	// TODO: Switch using ConfigMap
-	return true
+	// Return false if configmap not present/any errors
+	vpcCniConfigMap, err := c.K8sAPI.GetConfigMap(config.VpcCniConfigMapName)
+
+	if err == nil && len(vpcCniConfigMap.Data) > 0 {
```

Comment:
shouldn't this be `vpcCniConfigMap.Data != nil`? We will need UTs to test all of these conditions